src/System.Collections.Concurrent/dir.props

```diff
@@ -2,6 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.14.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
I talked with @ericstj and it looks like we intentionally keep AssemblyVersion as it was when we are adding API for core-only. So in this case, I guess that we have to revert this change, and that might solve your build issue.